locale/pt-br/blog/video/bryan-cantrill-instrumenting-the-real-time-web.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Bryan Cantrill: Instrumenting the Real Time Web"
```

Comment:
Should these title values be translated as well?

Reply:
yes, I will do it, thanks @nschonni